libctru/include/3ds/services/mic.h

```diff
@@ -4,95 +4,125 @@
  */
 #pragma once
 
-//See also: http://3dbrew.org/wiki/MIC_Services
+/// Microphone audio encodings.
+typedef enum
+{
+	ENCODING_PCM8 = 0,         ///< Unsigned 8-bit PCM.
+	ENCODING_PCM16 = 1,        ///< Unsigned 16-bit PCM.
+	ENCODING_PCM8_SIGNED = 2,  ///< Signed 8-bit PCM.
+	ENCODING_PCM16_SIGNED = 3, ///< Signed 16-bit PCM.
+} MIC_Encoding;
+
+/// Microphone audio sampling rates.
+typedef enum
+{
+	SAMPLE_RATE_32730 = 0, ///< 32730 Hz
+	SAMPLE_RATE_16360 = 1, ///< 16360 Hz
+	SAMPLE_RATE_10910 = 2, ///< 10910 Hz
+	SAMPLE_RATE_8180 = 3,  ///< 8180 Hz
```

Comment:
Those enums should be anonymous, and their identifiers should be prefixed with MICU (e.g. `MICU_ENCODING_PCM8`, `MICU_SAMPLERATE_32730`).